.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    name: Smoke Test
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+          cache: 'npm'
+      - run: npm install
+      - name: Login
+        run: ./bin/run login --api-key ${{ secrets.USER_API_KEY_TEST }} --account-id ${{ secrets.USER_ACCOUNT_TEST }}
```

Comment:
This is a nit pick but is there way we can link ./bin/run to checkly-cli so it looks prettier :D

Reply:
Something like example below but we need to rename run or give it an alias

echo "::add-path::$RUNNER_WORKSPACE/$(basename $GITHUB_REPOSITORY)/bin" # Make it accessible from runner

Reply:
We could just add a symlink to `/bin/run` as `/bin/checkly-cli` to the bin directory. That way we can use it here with your `add-path` workaround and in dev as `checkly-cli` more easily as well. What do you think?